CHANGELOG.md

```diff
@@ -1,190 +1,173 @@
 # Changelog
 
-## 2.9.2 (2025-02-11)
+All notable changes to this project will be documented in this file.
 
-### Fixes
-- Get NDK version from the `source.properties` file when uploading NDK symbol files [172](https://github.com/bugsnag/bugsnag-cli/pull/172)
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```

Comment:
I don't think we really need/want this preamble? IMO it doesn't require explanation.